tests/Unit/Libraries/Cms/Schema/PostgresqlChangeItemTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * @package     Joomla.UnitTest
+ * @subpackage  Schema
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Tests\Unit\Libraries\Cms\Schema;
+
+use Joomla\CMS\Schema\ChangeItem\PostgresqlChangeItem;
+use Joomla\Database\Pgsql\PgsqlDriver;
```

Comment:
@laoneo I have a question. In the file for MySQL I had to use the "MysqliDriver" because only that implements the utf8mb4 specific methods which we unfortunately still need. So for consistency I've decided to use the "PgsqlDriver" here. But I think that's not really necessary. The general "DatabaseDriver" should work as well. Should I change it here do use that?